tfrecorder/types.py

```diff
@@ -46,7 +46,7 @@ class ImageUri(SupportedType):
 class SplitKey(SupportedType):
   """Supports split key columns."""
   feature_spec = tf.io.FixedLenFeature([], tf.string)
-  allowed_values = ['TRAIN', 'VALIDATION', 'TEST', 'DISCARD']
+  allowed_values = ['train', 'validation', 'test', 'discard']
```

Comment:
You should also update any code that uses `DISCARD` and replace it with `discard`.

Reply:
@cdibona I edited the files too, but one running some tests. I am getting this error.
`    raise NotImplementedError(_DECODE_DEPRECATION_MESSAGE)
NotImplementedError: TFXIO should be used to decode CSV.`
Can you help me out? I couldn't find help on it online.